.circleci/config.yml

```diff
@@ -58,7 +58,7 @@ workflows:
           bot-token-variable: GHI_TOKEN
           bot-user: cpe-bot
           fail-if-semver-not-indicated: true
-          publish-version-tag: false
+          publish-version-tag: true
```

Comment:
I think we need to specify `ssh-fingerprints` too if `publish-version-tag` is set to true

Reply:
We don't. I do the same thing with the Hugo orb. It works fine: https://github.com/CircleCI-Public/hugo-orb/blob/master/.circleci/config.yml

The fingerprints are only needed when you have multiple keys so you can specify a specific one.

Reply:
The fingerprints are needed unless perhaps you used a user key on the hugo orb repo. By default, you will not have write permissions. The ssh keys are for a write deploy key

Reply:
The fingerprints are not needed. If you follow the link for the Hugo orb, I don't use them there and pushing Git tags works just fine.

Whether you specify a fingerprint or not, a GitHub user key **is needed** and I've already added it to this project.

Reply:
I see this repo has a user key, and so the fingerprints should not be needed. Most orbs are not configured this way.

Reply:
Can you show me one? Most of our orbs don't use Git tags at all. The few that do have a fingerprint I believe for the user key, but it's not needed. Just an extra few lines in the config. If they push a Git tag, they should have a user key though. I don't see another way.